app/lib/features/search/widgets/quick_actions_builder.dart

```diff
@@ -55,16 +55,16 @@ class QuickActionsBuilder extends ConsumerWidget {
                   label: const Text('Update'),
                 )
               : null,
-          isActive(LabsFeature.tasks)
-              ? OutlinedButton.icon(
-                  onPressed: () {
-                    navigateTo(route: Routes.actionAddTask, push: true);
-                    debugPrint('Add Task');
-                  },
-                  icon: const Icon(Atlas.plus_circle_thin),
-                  label: const Text('Task'),
-                )
-              : null,
+          // isActive(LabsFeature.tasks)
```

Comment:
why its commented out, we don't want to have tasks being accessed from quick actions ?

Reply:
Okay then we can remove that and re-add when its done ;)